Automattic/jetpack
# Social Menu, Social Media Icons: update Patreon logo

## Proposed changes:

Patreon's logo is changing: https://news.patreon.com/articles/patreon-reimagined
Our logo should change too.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to `yoursite.com/wp-content/plugins/jetpack/_inc/social-logos/social-logos.html`
* Check the new logos
* Go to Jetpack > Settings and enable extra sidebar widgets.
* Switch to an old theme, like Twenty Ten.
* Go to Plugins > Add New and install the Classic Widgets plugin.
* In a functionality plugin, add the following:
```php
// Add social menu support
add_action( 'after_setup_theme', function () {
	add_theme_support( 'jetpack-social-menu', 'svg' );
} );
```
* Edit your theme and add the following, in `footer.php` for example
```php
<?php jetpack_social_menu(); ?>
```
* Go to Appearance > Customize > Menus
* Create a new menu with some links, e.g. `https://www.facebook.com/jetpackme` and `https://patreon.com/user?u=2945881`
* Assign that menu to the Social Menu location
* Go to Appearance > Widgets menu
* Drag the Social Icons widget to one of the active widget areas, and add some links again.
* Check how things look like on the frontend, for both your social menu and the Social Icons Widget you created.
* Go to Jetpack > Settings > Sharing
* Enable the sharing buttons
* Go to Settings > Sharing and enable some buttons.
* Ensure the icons look good on the frontend as well.
